spec/date.html

```diff
@@ -420,6 +422,18 @@ <h1>CreateDate ( _y_, _m_, _d_ [ , _newTarget_ ] )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-createdatefrominstance" aoid="CreateDateFromInstance">
```

Comment:
If we decide to go with https://github.com/tc39/proposal-temporal/pull/429, then this should be named `CreateTemporalDateFromInstance`.